### ISSUE TITLE
Release package

### ISSUE THREAD

**Issue by torydebra** [alexandrefch/ros-image-transport-py#1]
Hi, 

thanking you for this contribution, I would like to ask if you plan to release it as binary 
https://wiki.ros.org/ROS/ReleasingAPackage

**Comment by alexandrefch** [comment]
Hi, thank for your interest !

I never thought about it because Noetic is now an old ROS version and I saw that now in ROS 2, there is a native python binding of official image_transport from ros-perception. [ROS 2 image_transport_py](https://github.com/ros-perception/image_common/tree/rolling/image_transport_py)

**Comment by torydebra** [comment]
yes, I saw there is the ros2 version but some people still rely on the ros1 :/ (at least for a while), so I was asking
